sema4ai/src/sema4ai_code/action_server.py

```diff
@@ -101,7 +102,7 @@ def __init__(self, action_server_location: str):
 
     def _run_action_server_command(
         self,
-        args: list[str],
+        args: List[str],
```

Comment:
Why the change here? 

i.e.: Python 3.10 is currently a requirement, so, `list` should usually be preferred over `List`...